pom.xml

```diff
@@ -0,0 +1,621 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <name>Closure Templates</name>
+  <groupId>com.google.template</groupId>
+  <artifactId>soy</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
```

Comment:
When the move to GitHub was announced, it was labelled "2.4" [1], so I think we should use 2.5-SNAPSHOT here. I'm a bit concerned that this is likely to compare lower than the current `2012-12-21` release on Central (if I read [the rules](https://maven.apache.org/ref/3.2.3/maven-artifact/apidocs/org/apache/maven/artifact/versioning/ComparableVersion.html) correctly).
So we might have to change the `groupId` and/or `artifactId` too (if we don't, then if you depend transitively on both `2012-12-21` and `2.5` then Maven –or Gradle, or Ivy– would choose the `2012-12-21` version when you'd want it/them to pick the `2.5` one).
As an alternative, we could use `9999-SNAPSHOT` and continue to use the timestamp scheme for releases.

[1] https://groups.google.com/d/msg/closure-templates-discuss/6Ykccvp5uJ0/BdP0dMfhTHoJ